entities/dashboard/staffWidgetsBoard.json

```diff
@@ -0,0 +1,115 @@
+{
+    "type": "object",
+    "properties": {
+      "uid": {
+        "type": "string",
+        "description": "Unique identifier of the staffWidgetsBoard object"
+      },
+      "board_layout": {
+        "type": "string",
+        "description": "The CSS class of the layout. This will determine the layout of the widgets dashboard based on the class.",
+        "enum": [
+          "dashboard"
+        ]
+      },
+      "type": {
+        "type": "string",
+        "description": "the grid frame of the layout, determines the layout frame containing the board",
+        "enum": [
+          "2*2",
```

Comment:
wrong enum

Reply:
This what was discussed and approved with Views team, name and values

Reply:
This is different from my requirements. I do not approve it. Let's take it back to views.